src/components/evaluation/EvaluationTable.vue

```diff
@@ -397,7 +397,11 @@ export default {
       return this.$currentUser.isAdmin || !this.isSelectedTermLocked
     },
     rowValid() {
-      return this.rules.currentTermDate(this.selectedStartDate) === true
+      const sisDates = this.evaluations
+        .map(e => e.id === this.editRowId ? [e.courseStartDate, e.courseEndDate] : '')
+        .filter(e => !!e)
```

Comment:
@cesarvh For purposes of finding the object in the evaluations array matching `this.editRowId`, Lodash `find` might be a more efficient and readable than mapping to a transient array: https://lodash.com/docs/4.17.15#find